ledger/participant-state/kvutils/src/main/protobuf/com/daml/ledger/participant/state/kvutils/store/state.proto

```diff
@@ -57,6 +57,9 @@ message DamlCommandDedupValue {
   // the time until when future commands with the same
   // deduplication key will be rejected due to a duplicate submission
   google.protobuf.Timestamp deduplicated_until = 2;
+  //The submission time of the initial command
+  // We use submission time because record time is not available during pre-execution
+  google.protobuf.Timestamp submission_time = 3;
```

Comment:
Nit (I don't think we use this protos to genarate docs btw though):

```suggestion
  // Tthe time until when future commands with the same
  // deduplication key will be rejected due to a duplicate submission.
  google.protobuf.Timestamp deduplicated_until = 2;
  // The submission time of the initial command.
  // We use submission time because record time is not available during pre-execution.
```

Reply:
`Tthe` -> `the`